app/src/main/res/layout/fragment_manage_site_permissions_feature_phone.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:layout_marginTop="@dimen/radio_button_preference_vertical">
+
+    <RadioGroup android:layout_width="match_parent"
+                android:layout_height="wrap_content">
```

Comment:
One nit: Tab size on some of these files are 8 spaces instead of 4

Reply:
Thanks I will fix it!

Reply:
Fixed